Motorcycle.js

```diff
@@ -1,6 +1,20 @@
 var Car = require('./Car.js');
 
 //Implement your Motorcycle here
-function Motorcycle() {};
+function Motorcycle(make, model, year, color, passengers) {
```

Comment:
Then, you can use an if statement to override the amount of seats a user enters if it's greater than 2:
```
if (this.seats > 2) {
    this.seats = 2;
}
```